### ISSUE TITLE
Does support YOLOv5 4.0?

### ISSUE THREAD

**Issue by pango99** [linghu8812/tensorrt_inference#42]
Does support YOLOv5 4.0?

**Comment by linghu8812** [comment]
yes, it supports YOLOv5 4.0 now and support yolov5s6.pt and so on in YOLOv5 4.0.

just run with
```
./yolov5_trt ../config6.yaml ../samples
```

**Comment by pango99** [comment]
Great,thank you very much for your sharing

**Comment by chandu1263** [comment]
Where can I find yolov5s6.yaml?

**Comment by linghu8812** [comment]
@chandu1263 
https://github.com/ultralytics/yolov5/blob/master/models/hub/yolov5s6.yaml

**Comment by chandu1263** [comment]
I am not able to convert yolov5s6 to onnx.
Facing this issue
`ONNX export failure: Exporting the operator silu to ONNX opset version 10 is not supported. Please open a bug to request ONNX export support for the missing operator.`
I tried opset versions 10-12, but couldn't solve it.
Any help would be great!

**Comment by linghu8812** [comment]
@chandu1263 
check these lines
https://github.com/linghu8812/tensorrt_inference/blob/5f22509ead8c5b3432b682dde956d1cbcae0137c/yolov5/export_onnx.py#L39-L40

**Comment by chandu1263** [comment]
Thanks, it works. I have been using some other export_onnx.py file.
